pyveda/vedaset/stream/vedastream.py

```diff
@@ -146,6 +146,20 @@ def __init__(self, mltype, classes, count, gen, partition, bufsize,
         if auto_startup:
             self._start_consumer()
 
+    @classmethod
+    def from_vc(cls, vc, count=None, bufsize=50, cachetype=collections.deque,
+                auto_startup=False, auto_shutdown=False, fetcher=None, loop=None):
+        if count == None:
```

Comment:
so people say that `==` should be reserved for doing things like comparing equality between things of similar type, like numbers. When we wanna check for `None`, the pythonic way is to check the instance: `if count is None: ...`
or, even MORE pythonic, `if not count: ...`. What's cool about that last one is that that expression will also evaluate to False if somebody passes in count=0, which would be dumb